tools/topology/m4/dai.m4

```diff
@@ -178,7 +178,6 @@ define(`DAI_ADD',
 `undefine(`DAI_BUF')'
 `undefine(`DAI_PERIODS')'
 `undefine(`DAI_FORMAT')'
-`undefine(`SCHEDULE_FRAMES')'
```

Comment:
Ok, this should be merged into previous patch, we also need to mark the token SCHEDULE_FRAMES as deprecated in FW and kernel headers (dont delete as it will be ignored now) and increment MINOR ABI.

Reply:
well, with review of the last version Liam was asking for it :)

Reply:
@xiulipan and yes the topology setting is removed, but I set it to always 0 on the actual pipeline struct